### PR TITLE
Chao/export image

### PR DESCRIPTION
**Pull request by AsuraZeng**

### CODE REVIEW

**Comment by BaochengSu** [comment]
I think the final goal for firmware update deploying is that the firmware update package is generated by the build and copied into deploy, not only the .deb for the updating tool.

However, this could be the first step towards the final goal.

**Review by BaochengSu — APPROVED** [review]

File: recipes-bsp/u-boot/u-boot-iot2050.inc [hunk 1]
```diff
@@ -95,6 +95,7 @@ dpkg_build_sstate_finalize_append() {
 
 do_deploy() {
     cp -f "${S}/${SPI_FLASH_IMG}" "${DEPLOY_DIR_IMAGE}/${SPI_FLASH_DEPLOY_IMG}"
+    cp -f "${S}/u-boot-initial-env" "${DEPLOY_DIR_IMAGE}/"
```
Comment (jan-kiszka) [inline recipes-bsp/u-boot/u-boot-iot2050.inc:98]: You also need to update sstate caching.

Comment (AsuraZeng) [reply recipes-bsp/u-boot/u-boot-iot2050.inc:98]: HI jan
I have no expericing how to modify this.
and for the deploy stage, do there need to add to the sstate caching?

Comment (jan-kiszka) [reply recipes-bsp/u-boot/u-boot-iot2050.inc:98]: Just study the code a few lines above and/or the commit that added the already existing line. And, yes, it's mandatory to not break local builds.

Comment (AsuraZeng) [reply recipes-bsp/u-boot/u-boot-iot2050.inc]: has modified, pls help to review

**Review by jan-kiszka — APPROVED** [review]

**Review by BaochengSu — APPROVED** [review]

**Pull request merged**